### PR TITLE
Add news about GCC2019 training nomination

### PR DESCRIPTION
**Pull request by bebatut**
The nomination of training will open on the 1st of december

### CODE REVIEW

File: src/events/gcc2019/faq/index.md [hunk 1]
```diff
@@ -32,7 +32,9 @@ Nominated topics can cover a wide range. For example:
 
 This list only shows some examples. If you think the communities would be interested in a topic, then please nominate it!  And if you are looking for ideas, see the topic nominated in: [2016](bit.ly/gcc2016noms), [2015](bit.ly/gcc2015vote), [2014](bit.ly/1s6NtMN), [2013]( bit.ly/1i2j1gN) and the [Galaxy Events page](galaxyproject.org/events/).
 
-The training nomination will open on the **December, 10** and close it on the **December, 31**. The voting process will then go from the **January, 15** to the **January, 31st**.
+The training nomination is open from the **December, 1st** to the **December, 31**. Topics will be compiled into a uniform list by the GCC2017 Organizing Committee, and topics will be voted on by the Galaxy Community starting from the **January, 15th** to the **January, 31st**.
```
Comment (erasche) [inline src/events/gcc2019/faq/index.md]:
```suggestion
The training nomination is open from **December, 1st** to **December, 31**. Topics will be compiled by the GCC2017 Organizing Committee, and voted on by the Galaxy Community from **January, 15th** to **January, 31st**.
```

File: src/news/gcc2019_training_nomination/index.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
+---
+title: 'Nominate a training for the 2019 Galaxy Community Conference (GCC2019)'
+date: '2018-12-01'
+---
+<img class="img-fluid mx-auto" src="/src/news/gcc2019_training_nomination/training_gcc_logo_blue_bg.png" alt="GCC2019" />
+
+The [**2019 Galaxy Community Conference (GCC2019)**](/src/events/gcc2019/index.md) will be held in be held in **Freiburg, Germany, 1-6 July**. Like [previous Galaxy Community Conferences](/src/gcc/index.md), GCC2019 will feature invited keynotes, accepted talks, posters, demos, birds-of-a-feather gatherings and **training**. But wThe format of GCC this year will be a bit different than the previous years: it starts with 1 day of training and then 3 days with more specialized training sessions aligned with talks. *You can read more about the new format on the [conference FAQ](/src/events/gcc2019/faq/#what-will-be-the-format-for-gcc-in-2019).*
```
Comment (erasche) [inline src/news/gcc2019_training_nomination/index.md]:
```suggestion
The [**2019 Galaxy Community Conference (GCC2019)**](/src/events/gcc2019/index.md) will be held in be held in **Freiburg, Germany, 1-6 July**. Like [previous Galaxy Community Conferences](/src/gcc/index.md), GCC2019 will feature invited keynotes, accepted talks, posters, demos, birds-of-a-feather gatherings and **training**. But this year the format of GCC will be quite different to the previous years: it starts with 1 day of training and then 3 days of more specialized training sessions intermixed with talks. *You can read more about the new format on the [conference FAQ](/src/events/gcc2019/faq/#what-will-be-the-format-for-gcc-in-2019).*
```

File: src/news/gcc2019_training_nomination/index.md [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+
+**The training topics that are offered are determined by *the community* so *[BY YOU!](https://docs.google.com/forms/d/1fBPjEmw7Td52rhBnC0SvflnQCD7Ue5IMJ9RkM0SjnYg/prefill)***
+
+Please give some thought to what topics you would like to see offered, or that you would like to offer, and then [submit them as topic nominations](https://docs.google.com/forms/d/1fBPjEmw7Td52rhBnC0SvflnQCD7Ue5IMJ9RkM0SjnYg/prefill). 
+
+Nominated topics can cover a wide range. For example:
+- Introduction to Using Galaxy
+- Scientific topic oriented trainings
+- Community specific trainings
+- Development and administration around Galaxy
```
Comment (erasche) [inline src/news/gcc2019_training_nomination/index.md]:
```suggestion
- Galaxy development and administration
```

File: src/news/gcc2019_training_nomination/index.md [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+- Train the trainers
+
+This list only shows some examples. **Any topic of interest to the Galaxy Community can be nominated and you are encouraged to nominate more than one topic.** If you are looking for ideas, see the topic nominated in: [2016](bit.ly/gcc2016noms), [2015](bit.ly/gcc2015vote), [2014](bit.ly/1s6NtMN), [2013]( bit.ly/1i2j1gN) and the [Galaxy Events page](galaxyproject.org/events/).
```
Comment (erasche) [inline src/news/gcc2019_training_nomination/index.md]:
```suggestion
This list only shows some examples. **Any topic of interest to the Galaxy Community can be nominated and you are encouraged to nominate more than one topic.** If you are looking for ideas, see the topics nominated in: [2016](bit.ly/gcc2016noms), [2015](bit.ly/gcc2015vote), [2014](bit.ly/1s6NtMN), [2013]( bit.ly/1i2j1gN) and the [Galaxy Events page](galaxyproject.org/events/).
```

File: src/news/gcc2019_training_nomination/index.md [hunk 5]
```diff
@@ -0,0 +1,24 @@
...
+
+The training nomination is open from the **December, 1st** to the **December, 31**. Topics will be compiled into a uniform list by the GCC2017 Organizing Committee, and topics will be voted on by the Galaxy Community starting from the **January, 15th** to the **January, 31st**.
```
Comment (erasche) [inline src/news/gcc2019_training_nomination/index.md]:
```suggestion
The training nomination is open from **December, 1st** to **December, 31**. Topics will be compiled by the GCC2017 Organizing Committee, and voted on by the Galaxy Community from **January, 15th** to **January, 31st**.
```

(copied from above)

File: src/news/gcc2019_training_nomination/index.md (hunk 2 above)
Comment (bgruening) [inline src/news/gcc2019_training_nomination/index.md]: "will be held in be held" -> will be in Freiburg

Comment (erasche) [reply src/news/gcc2019_training_nomination/index.md]: Updated my suggested change to incorporate.

**Pull request merged**